certified-connectors/Litera/apiDefinition.swagger.json

```diff
@@ -0,0 +1,198 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Litera Search",
+		"version": "1.0",
+		"description": "The Litera Search connector exposes functions to execute complex natural language matter searches, giving legal professionals the ability to quickly find and drill down into the details of their firm's litigation and transaction experience.  The connector is a go-to tool for legal insights and information management.",
+		"contact": {
+			"name": "Litera Customer Care",
+			"url": "https://www.litera.com/support",
+			"email": "support@litera.com"
+		},
+		"x-ms-api-annotation": {
+			"status": "Preview"
+		},
+		"x-ms-keywords": [
+			"Litera",
+			"GetMatterList",
+			"GetMatterNarrative",
+			"GetMatterNarratives",
+			"GetMatterDetail",
+			"GetMatterDetails",
+			"matter list",
+			"list of matters",
+			"matters",
+			"matter",
+			"deals",
+			"deal",
+			"cases",
+			"case",
+			"legal matters",
+			"legal matter",
+			"legal cases",
+			"legal case",
+			"legal deals",
+			"legal deal",
+			"transactions",
+			"transaction",
+			"matter narrative",
+			"matter overview",
+			"matter description",
+			"description for matter",
+			"narrative for matter",
+			"overview for matter",
+			"description of matter",
+			"narrative of matter",
+			"overview of matter"
+		]
+	},
+	"host": "copilotdevappservice.azurewebsites.net",
+	"basePath": "/",
+	"schemes": [
+		"https"
+	],
+	"consumes": [],
+	"produces": [],
+	"paths": {
+		"/GetMatterList": {
+			"get": {
+				"tags": [
+					"Matter List"
+				],
+				"responses": {
+					"200": {
+						"description": "default",
+						"schema": {}
+					}
+				},
+				"produces": [
+					"application/json"
+				],
+				"summary": "Allows the user to search for legal matters and their associated IDs.",
```

Comment:
'summary' value must end with alphanumeric character. ValuePath: GetMatterList/get/summary

Reply:
done - will be reflected in next commit.